packages/cli/src/Init.ts

```diff
@@ -110,6 +131,9 @@ export class Init implements Command {
     
              -h, --help   Display this help message
   --datasource-provider   Define the datasource provider to use: postgresql, mysql, sqlite, sqlserver, mongodb or cockroachdb
+   --generator-provider   Define the generator provider to use: by default, it will be prisma-client-js
+      --preview-feature   Define a preview feature to use: by default, there will be none
+               --output   Define the output path to use: by default, it will be node_modules/.prisma/client
```

Comment:
Now actually looking at it in context, the `:` is to be used to list the options.
The default output is also implicit, so I suggest to change that wording as well:

```suggestion
   --generator-provider   Define the generator provider to use (by default, it will be prisma-client-js)
      --preview-feature   Define a preview feature to use (by default, there will be none)
               --output   Define the output path to use (by default, it will not be defined)
```